src/Components/Web.JS/src/InputLargeTextArea.ts

```diff
@@ -0,0 +1,32 @@
+import { DotNet } from '@microsoft/dotnet-js-interop';
+
+export const InputLargeTextArea = {
+  init,
+  getText,
+  setText,
+  enableTextArea,
+};
+
+function init(callbackWrapper: any, elem: HTMLTextAreaElement): void {
+  elem.addEventListener('change', function(): void {
```

Comment:
Noob question - does it work as expected with things like `preventDefault` / `stopPropogation`?

Reply:
Not sure I follow. Asking if this component still works if the following is present elsewhere in the client side JS code?

```
document.getElementById("textareaComponent").addEventListener("click", function(event){
  event.preventDefault()
});
```

Reply:
I was thinking of https://docs.microsoft.com/en-us/aspnet/core/blazor/components/event-handling?view=aspnetcore-5.0#stop-event-propagation, but that doesn't make sense here. Oh well, let's ignore this?